guides/release/templates/actions.md

```diff
@@ -114,15 +124,23 @@ export default class Post extends Component {
   @tracked title;
 
   @action
-  createPost() {
+  createPost(event) {
+    event.preventDefault();
     // Do something with `this.title`
   }
 }
 ```
 
+The native JavaScript form submission event is the sole argument to 
+`createPost`. The form submission event needs to have `preventDefault` 
+called on it in order to opt out of the browser-default behavior -- which, 
+for forms, is to POST to the current URL.
+
 ## Attaching Actions to Non-Clickable Elements
 
-Note that while Ember currently permits you to add an action to any DOM element, not all DOM elements are eligible to receive focus, according to HTML standards.
+Note that while Ember currently permits you to add an action to any DOM 
+element, not all DOM elements are eligible to receive focus, according to 
+HTML standards.
```

Comment:
This `Note` is pretty confusing. As a reader I'm not sure what this comment (something something focus) has to do with actions on non-clickable elements.

This is likely trying to nudge the user to think about accessibility? Can it be structured so that a first time using coming to this section can get the information they expected before that consideration distracts them?

Reply:
> This Note is pretty confusing.

yeah, I struggled with it a bit, too. 

> This is likely trying to nudge the user to think about accessibility?

I think so. I've primarily only used Chrome and FireFox, so I haven't run in to any issues with browsers not *letting* things be clickable.  I wonder if this text is counting a screen reader or other a11y tool as a browser.

Reply:
Not sure what to do about these notes -- kinda out of scope for this Pr, but if someone else wants to tackle it / open an issue for it, be my guest!